### PR TITLE
[CSFS-6] Style the Contact Us Page using CSS

### PR DESCRIPTION
**Pull request by Oge-nna**
## Task: `[CSFS-6] Style the Contact Us Page using CSS`
- `Created and styled a contact page`

ContactUs
ContactUs2
ContactUsMobile
ContactUsMobile2
ContactUsTab
ContactUsTab2

### CODE REVIEW

File: submissions/ogennaOgbogu/htmlForms/contactUs.html [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+                        </div>
+
+                        <div class="social-network">
+                            <p>CONNECT WITH US</p>
+                                <div class="social-img">
+                                    <img src="../HTML/Images/facebook_logo.png">
+                                <img src="../HTML/Images/gmail_logo.png">
+                                <img src="../HTML/Images/twitter_logo.png">
+                                <img src="../HTML/Images/youtube_logo.png">
+                                </div>
```
Comment (heisbabz) [inline submissions/ogennaOgbogu/htmlForms/contactUs.html:125]: Format this code block properly

File: submissions/ogennaOgbogu/htmlForms/contactUs.html [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+                                <br>
+                                <input type="text" class="subject" name="subject">
+                                <br>  
+                                <label for="message2">Message</label>
+                                <br>
+                                <textarea class="message2" name="message2"></textarea>
+                                <br>
+                                <button type="submit" class="submit-text">SEND</button>
+
+                    </form>
```
Comment (heisbabz) [inline submissions/ogennaOgbogu/htmlForms/contactUs.html:96]: Indent properly

File: submissions/ogennaOgbogu/CSS/contactUsStyleSheet.css [hunk 3]
```diff
@@ -0,0 +1,366 @@
...
+
+.message>p {
+    text-align: center;
+    margin: auto;
+    padding-bottom: 10px;
+}
+
+.form>input.full-name,.form>input.email,.form>input.subject {
+    width: 100%;
+    height: 0;
```
Comment (heisbabz) [inline submissions/ogennaOgbogu/CSS/contactUsStyleSheet.css:120]: You won't be able to type into your input box if you are setting the height to 0. I will advise that you remove this.

File: submissions/ogennaOgbogu/CSS/contactUsStyleSheet.css [hunk 4]
```diff
@@ -0,0 +1,366 @@
...
+
+html,body {
+    width: 100%;
+    height: 100%;
+    margin: 0;
+    padding: 0;
+    overflow-x: auto;
+    overflow-y: auto;
+}
+
```
Comment (heisbabz) [inline submissions/ogennaOgbogu/CSS/contactUsStyleSheet.css:13]: Remove extra space. There are other places where you have more than one blank line between your CSS selectors. Please ensure that you make the necessary changes.

File: submissions/ogennaOgbogu/CSS/contactUsStyleSheet.css [hunk 5]
```diff
@@ -0,0 +1,366 @@
...
+    width: 50%;
+    float: left;
+    margin: 0 20px;
+}
+
+.right-sidetab {
+    width: 50%;
+    text-align: end;
+    margin: auto;
+
```
Comment (heisbabz) [inline submissions/ogennaOgbogu/CSS/contactUsStyleSheet.css:39]: remove space here too

File: submissions/ogennaOgbogu/CSS/contactUsStyleSheet.css [hunk 6]
```diff
@@ -0,0 +1,366 @@
...
+.header {
+    background-color: rgba(000, 255, 000, 0.4);
+}
+
+nav {
+    display: flex;
+    margin: 0 10px;
+}
+
```
Comment (heisbabz) [inline submissions/ogennaOgbogu/CSS/contactUsStyleSheet.css:27]: extra space here too

File: submissions/ogennaOgbogu/CSS/contactUsStyleSheet.css [hunk 7]
```diff
@@ -0,0 +1,366 @@
...
+    text-align: center;
+}
+
+.border-left {
+    padding: 30px 50px;
+    white-space: nowrap;
+    margin: auto;
+}
+
+.phone-sign>p:nth-child(2) {
```
Comment (heisbabz) [inline submissions/ogennaOgbogu/CSS/contactUsStyleSheet.css:65]: Please avoid writing your CSS selectors together like this. It makes it difficult for others to read. Remember adding space in between. You can try `.phone-sign > p:nth-child(2)`

File: submissions/ogennaOgbogu/CSS/contactUsStyleSheet.css [hunk 8]
```diff
@@ -0,0 +1,366 @@
...
+    font-weight: bold;
+}
+
+.phone-sign>p>a {
```
Comment (heisbabz) [inline submissions/ogennaOgbogu/CSS/contactUsStyleSheet.css:69]: Same here too. Please ensure you make changes to other places where you might have this occurrence.

File: submissions/ogennaOgbogu/htmlForms/contactUs.html [hunk 9]
```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+
```
Comment (heisbabz) [inline submissions/ogennaOgbogu/htmlForms/contactUs.html:3]: Kindly format this page properly. Check out this [link](https://google.github.io/styleguide/htmlcssguide.html) to help you in styling your HTML and CSS pages.

**Review by heisbabz — CHANGES_REQUESTED** [review]

File: submissions/ogennaOgbogu/htmlForms/contactUs.html [hunk 10]
```diff
@@ -0,0 +1,133 @@
...
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Sign Up</title>
+        <link rel="stylesheet" type="text/css" href="../CSS/contactUsStyleSheet.css">
+        <link rel="preconnect" href="https://fonts.gstatic.com">
+        <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">     
+    </head>
+
+    <body>
+        
+
```
Comment (TundeOladejo) [inline submissions/ogennaOgbogu/htmlForms/contactUs.html:15]: Remove extra spaces

**Review by TundeOladejo — CHANGES_REQUESTED** [review]

File: submissions/ogennaOgbogu/htmlForms/contactUs.html (hunk 9 above)
Comment (Oge-nna) [reply submissions/ogennaOgbogu/htmlForms/contactUs.html:3]: Hi @heisbabz, I understand all the other changes except this one. Please explain what you want me to do, I'm confused.

Comment (TundeOladejo) [reply submissions/ogennaOgbogu/htmlForms/contactUs.html:3]: What am saying is that you should format it properly. The link I shared with you is the HTML/CSS style guide.

**Review by TundeOladejo — CHANGES_REQUESTED** [review]

File: submissions/ogennaOgbogu/htmlForms/contactUs.html (hunk 9 above)
Comment (Oge-nna) [reply submissions/ogennaOgbogu/htmlForms/contactUs.html:3]: I have read the guide, I still do not understand what you want me to do, so if you be so inclined, kindly explain exactly want you want me to do.

**Review by TundeOladejo — APPROVED** [review]

**Pull request merged**